languoids/tree/afro1255/semi1276/west2786/cent2236/nort3165/aram1259/west2815/west2763/malu1250/md.ini

```diff
@@ -1,19 +1,17 @@
 # -*- coding: utf-8 -*-
 [core]
-name = Ma'lula
+name = Maaloula
 level = dialect
 macroareas = 
 	Eurasia
 countries = 
 
 [altnames]
 multitree = 
```

Comment:
Changing language names inherited from multitree is generally not useful, because the purpose they serve is making clear that a certain language was named with these names in multitree. So that's a legacy we have to live with.

Reply:
Thank you for your comment.
All those alternative names exist because "Maaloula" (pronounced /Maʕlūla/) is an Aramaic name, not English, German etc. Keeping alternatives is not a problem, since as you said they are there because this is how some scholars used them.
However, "Maaloula" is the closest name to the original one and the most used name, so I suggest to use it as an official name of the language, in order to protect the identity and help the new scholars not to be confused.